pro/src/components/pages/Offers/Offers/OfferItem/Cells/DeleteDraftCell/DeleteDraftCell.tsx

```diff
@@ -42,19 +42,19 @@ const DeleteDraftCell = ({ offer, refreshOffers }: IDeleteDraftOffers) => {
 
   return (
     <>
-      {isConfirmDialogOpen && (
-        <ConfirmDialog
-          icon={TrashIcon}
-          cancelText="Annuler"
-          confirmText="Supprimer ce brouillon"
-          onCancel={closeDeleteDraftDialog}
-          onConfirm={onConfirmDeleteDraftOffer}
-          title={`Voulez-vous supprimer le brouillon : "${offer.name}"`}
-        >
-          <></>
-        </ConfirmDialog>
-      )}
       <td className={styles['draft-column']} align="right">
+        {isConfirmDialogOpen && (
+          <ConfirmDialog
+            icon={TrashIcon}
+            cancelText="Annuler"
+            confirmText="Supprimer ce brouillon"
+            onCancel={closeDeleteDraftDialog}
+            onConfirm={onConfirmDeleteDraftOffer}
+            title={`Voulez-vous supprimer le brouillon : "${offer.name}"`}
+          >
+            <></>
+          </ConfirmDialog>
+        )}
```

Comment:
c'est pas équivalent d'écrire ça ?
le  <></> intutile me gène

```suggestion
        {isConfirmDialogOpen && (
          <ConfirmDialog
            icon={TrashIcon}
            cancelText="Annuler"
            confirmText="Supprimer ce brouillon"
            onCancel={closeDeleteDraftDialog}
            onConfirm={onConfirmDeleteDraftOffer}
            title={`Voulez-vous supprimer le brouillon : "${offer.name}"`}
          />
        )}
```

Reply:
Alors c'est pas excatement équivalent. Il faut faire une petite modif à ConfirmDialog pour ne pas avoir besoin de `children`